### DISCUSSION TITLE
cluster 集群节点安全问题

### DISCUSSION THREAD

**Discussion by JieTrancender** [cloudwu/skynet#1334] [Q&A] [answered]
我们集群使用cluster构建，最近发现一直有人尝试连接我们用于集群通信的端口，想通过clustername配置白名单方式限制，请问一下这样做会不会有什么问题？
![QQ20210126-141340](https://user-images.githubusercontent.com/13239914/105807341-bf327480-5fe0-11eb-9fb2-1ee80cffa2af.png)

**Comment by hanxi** [comment]
建议使用系统的防火墙，限定白名单 ip 才能互通。

**Reply by JieTrancender** [reply]
这个是可以的，但是加节点的时候时很麻烦，需要找机器提供方或者运维的。

**Comment by cloudwu** [comment, marked as answer]
在业务层做肯定不如专门的防火墙。即使不用系统级的防火墙，你也可以在 cluster 外套一个你自己的防火墙网关，转发到本地 cluster 端口。我觉得防火墙业务不应该加到 cluster 服务中。

**Reply by JieTrancender** [reply]
好的，谢谢。
